src/components/ToscaDataViewer/index.jsx

```diff
@@ -52,7 +52,7 @@ const ToscaDataViewer = (props) => {
         index={res._index}
         id={res._id}
       />
-      {browseUrl ? (
+      {(res.metadata.exists_in_s3_rs !== false && browseUrl) ? (
```

Comment:
i'd recommend changing this to:
```js
(res.metadata && res.metadata.exists_in_s3_rs !== false) && browseUrl
```
this is in case `res` doesn't have a field called `metadata`
but this might raise an issue if there exists a `browse_url` but `res` the document doesn't have a `metadata` field

i'd recommend not putting the `exists_in_s3_rs` in `metadata` but outside

Reply:
@DustinKLo Please check the latest update. I have a new variable hide_browse_url which will be true only when res.metadata and res.metadata.exists_in_s3_rs===false. Other case, it will be false. so should work for all cases